systemvm/patches/debian/config/opt/cloud/bin/merge.py

```diff
@@ -274,3 +277,46 @@ def __moveFile(self, origPath, path):
             os.makedirs(path)
         timestamp = str(int(round(time.time())))
         os.rename(origPath, path + "/" + self.fileName + "." + timestamp)
+
+
+class PrivateGatewayHack:
+
+
+    @classmethod
+    def update_network_type_for_privategateway(cls, dbag, data):
+        ip = data['router_guest_ip'] if 'router_guest_ip' in data.keys() else data['public_ip']
+
+        initial_data = cls.load_inital_data()
```

Comment:
Should we have a conditional around load_initial_data? according to lines 148 and 154 this is called twice.